server/protocols/webadmin/webadmin-mailrepository/src/test/java/org/apache/james/webadmin/routes/MailRepositoriesRoutesTest.java

```diff
@@ -406,19 +417,26 @@ public void retrievingRepositorySizeShouldReturnNumberOfContainedMails() throws
     @Test
     public void retrievingAMailShouldDisplayItsInformation() throws Exception {
         when(mailRepositoryStore.get(URL_MY_REPO)).thenReturn(Optional.of(mailRepository));
-
+        SimpleDateFormat dateTimeFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
when it comes to Dates, you should rather use java.time package.
In this case, It looks like you want ISO_INSTANT from `DateTimeFormatter`

Reply:
The TimeZone is missing but I take the idea